backend/build.gradle

```diff
@@ -94,3 +97,10 @@ task integrationTest(type: Test) {
     classpath = sourceSets.integrationTest.runtimeClasspath
     outputs.upToDateWhen { false }
 }
+
+docker {
+    springBootApplication {
+        baseImage = 'openjdk:8-alpine'
```

Comment:
What are the terms on that? Are we allowed to build derivative works?

Reply:
OpenJDK is GPLv2 licensed... 🤔

Reply:
The Docker base image used for the Spring Boot application is [here](https://github.com/docker-library/openjdk) MIT licensed. From [here](https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project) both are permissive, so we can relicense our derivative work however we want (ie. with Apache-2.0).

Reply:
Thanks for checking them all :)